packages/babel-preset-default/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## Enhancements
+
+-   Use `React` as the default scope variable for JSX pragma instead of `@wordpress/element`.
```

Comment:
Typography nit: when referring to the package name, putting it in backtick as a literal identifier, it should be lowercase `react`. There are multiple instances of this in the PR.